src/stories/Library/Lists/list-reservation/list-reservation.scss

```diff
@@ -93,10 +89,15 @@ $list-reservation-space-desktop: 24px;
 
   p {
     margin-top: 2px;
+    font-size: 14px;
+    margin-bottom: 16px;
   }
```

Comment:
We have talked about using our spacing variables in general. Why not using them here?

Reply:
Changed